modules/dtg/manifests/backup.pp

```diff
@@ -76,20 +76,27 @@
 # The name of the backup will be used as the directory name for the subdir containing the backups
 # user is the user to ssh in as
 # host is the host to ssh into
-define dtg::backup::hostsetup($user, $host, $weekday) {
+define dtg::backup::hostsetup($user, $group, $host, $weekday) { #needs to have a configurable backup group + default
```

Comment:
This doesn't have a default set, the user always has to specify something. Dealing with the case where it is set to undef is not the right way. Look at some of the other examples where we set the default. Syntax for this is similar to python.